tests/menu_items_registry.test.ts

```diff
@@ -143,14 +149,12 @@ describe("Menu Item actions", () => {
     env.clipboard.writeText = jest.fn(() => Promise.resolve());
     doAction(["edit", "copy"], env);
     expect(dispatch).toHaveBeenCalledWith("COPY");
-    expect(env.clipboard.writeText).toHaveBeenCalledWith(env.model.getters.getClipboardContent());
   });
 
   test("Edit -> cut", () => {
     env.clipboard.writeText = jest.fn(() => Promise.resolve());
     doAction(["edit", "cut"], env);
     expect(dispatch).toHaveBeenCalledWith("CUT");
-    expect(env.clipboard.writeText).toHaveBeenCalledWith(env.model.getters.getClipboardContent());
```

Comment:
Don't delete this, check that `env.clipboard.write` has been caleld with the correct value